### ISSUE TITLE
modifies MainActivity on android 

### ISSUE THREAD

**Issue by mazzinie** [henriquearthur/flutter_native_splash#45]
Android projects created in flutter 1.12 no longer use MainActivity.java generating a white screen before the home screen.

**Comment by camillo777** [comment]
Hello,
I have used this tool and I think I have got the same problem as you.
MainActivity is kotlin code.

Between the splash screen and the Flutter home screen I have got a **black screen** for a couple of seconds.

```
[✓] Flutter (Channel stable, v1.12.13+hotfix.7, on Mac OS X 10.14.6 18G3020, locale en-IT)
    • Flutter version 1.12.13+hotfix.7 at /Users/camillo/flutter
    • Framework revision 9f5ff2306b (4 days ago), 2020-01-26 22:38:26 -0800
    • Engine revision a67792536c
    • Dart version 2.7.0

[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/camillo/MyCamillo/Development/mobile/android/android-sdk-macosx/
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/camillo/MyCamillo/Development/mobile/android/android-sdk-macosx/
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C504
    • CocoaPods version 1.8.4

[!] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.41.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.7.1

[✓] Connected device (1 available)
    • SM G955F • ce0517153d937d0b02 • android-arm64 • Android 9 (API 28)
```

**Comment by Bharat1509** [comment]
Same issue i am also facing **black screen for a couple of seconds** 
is there any solution??

**Comment by skystar7** [comment]
Is this a bug with the package, or with the new version of Flutter?

**Comment by edufolly** [comment]
@Bharat1509 @skystar7 Probably this is the solution: [#54](https://github.com/henriquearthur/flutter_native_splash/issues/54#issuecomment-589237135)

**Comment by jonbhanson** [comment]
Should be resolved by 48981828e831768c4ad89a35475e8f9f24e0aca5.
